config/initializers/devise.rb

```diff
@@ -18,7 +18,7 @@
   # Configure the e-mail address which will be shown in Devise::Mailer,
   # note that it will be overwritten if you use your own mailer class
   # with default "from" parameter.
-  config.mailer_sender = "#{ENV["SYSTEM_EMAIL"]}"
+  config.mailer_sender = "#{ENV["SMTP_USERNAME"]}"
```

Comment:
I think this should also be `SMTP_FROM_EMAIL`? Can consider/fix later since i think the one in our mailers is overriding it (according to the comment above this line).